### PR TITLE
Add script to show org stats

### PR DESCRIPTION
**Pull request by Fryguy**
@chessbyte Please review.

Example output (trimmed for brevity)

```
$ bin/show_org_stats.rb --since "Jul 1, 2021"
...
Total Commits:
     335  Adam Grare  <adam@grare.com>
     157  Jason Frey  <fryguy9@gmail.com>
     139  Brandon Dunne  <brandondunne@hotmail.com>
     137  Keenan Brock  <keenan@thebrocks.net>
      91  Joe Rafaniello  <jrafanie@gmail.com>
...
Grand Total Commits: 1570

Total Merges:
     378  Jason Frey  <fryguy9@gmail.com>
     243  Adam Grare  <adam@grare.com>
      78  Kavya Nekkalapu  <37085529+kavyanekkalapu@users.noreply.github.com>
      77  Brandon Dunne  <bdunne@redhat.com>
      46  Joe Rafaniello  <jrafanie@users.noreply.github.com>
...
Grand Total Merges: 924
```

### CODE REVIEW

File: bin/show_org_stats.rb [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+  def run
+    ManageIQ::Release.each_repo(**@opts) { |repo| run_one(repo) }
+
+    puts
+    puts "Total Commits:"
+    print_totals(@total_commits)
+
+    puts
+    puts "Grand Total Commits: #{@total_commits.values.sum}"
```
Comment (chessbyte) [inline bin/show_org_stats.rb:33]: not necessarily for this PR, but would be nice to add number of different PR authors

File: bin/show_org_stats.rb [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+    puts
+    puts "Total Merges:"
+    print_totals(@total_merges)
+
+    puts
+    puts "Grand Total Merges: #{@total_merges.values.sum}"
```
Comment (chessbyte) [inline bin/show_org_stats.rb:40]: not necessarily for this PR, but would be nice to have similar stats on Issues Opened and Closed

File: bin/show_org_stats.rb (hunk 1 above)
Comment (Fryguy) [reply bin/show_org_stats.rb:33]: I updated the OP with example output

Comment (chessbyte) [reply bin/show_org_stats.rb:33]: thanks!

Comment (chessbyte) [reply bin/show_org_stats.rb:33]: is this counting commits or PRs?  Same question for the merge counts.

Comment (Fryguy) [reply bin/show_org_stats.rb:33]: The first half is commits, but not merges...the second half is merges, but not commits.  None of these are github stats; they are git stats.

Comment (Fryguy) [reply bin/show_org_stats.rb:33]: I can rename to show_org_git_stats.rb if that makes it clearer?

**Review by chessbyte — APPROVED** [review]
Merging as is - and we can improve it from here.
